backend/src/router/account/fungible-tokens.ts

```diff
@@ -20,7 +20,9 @@ type FungibleTokenMetadata = {
 };
 
 const base64ImageRegex = /^data:image/;
-const validateBase64Image = (base64Image: string | null): string | null => {
+export const validateBase64Image = (
+  base64Image: string | null
+): string | null => {
```

Comment:
I'd prefer to move this helper to utils, @luixo WDYT?

Reply:
Sure thing